packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java

```diff
@@ -70,6 +71,11 @@ public class ExtractionService {
 	private final ProvenanceService provenanceService;
 	private final CurrentUserService currentUserService;
 
+	// Used to get the Abstract text from PDF
+	private static final String NODE_TARGET = "Abstract";
+	private static final String NODE_PARAM = "detect_cls";
+	private static final String NODE_CONTENT = "content";
```

Comment:
I don't understand the choice of name here `NODE_XXX`?

Reply:
As we discussed, I will remove the target & param and leave the content definition